boards/DCM/Inc/App/App_Imu.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+#include <stdint.h>
+#include <stdbool.h>
+
+struct Imu;
+
+/** Creates Imu struct used for reading data from sensor
+ * @param update_data - pointer to function which updates data and returns true
+ * if update successful
+ * @param get_acceleration_x - pointer to function which returns x acceleration in m/s^2
+ * @param get_acceleration_y - pointer to function which returns y acceleration in m/s^2
+ * @param get_acceleration_z - pointer to function which returns z acceleration in m/s^2
+ * struct
+ * @return Imu data in ImuData struct
```

Comment:
The create struct doesn't return IMU data

Reply:
Yeah forgot to edit comments, it's changed now